taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -0,0 +1,92 @@
+import { Paper, Stack, TextField, Typography } from "@mui/material";
+import { useEffect, useState } from "react";
+
+const ListAllOtherProperties = ({ props }) => {
+    const [nodeObject, setNodeObject] = useState(null);
+    useEffect(() => {
+        setNodeObject(props);
+    }, [props])
+
+    // Stores 2 letter language code (LC) of the tags
+    let languageCode = ''; 
+    // Storing keys and values that needs to be rendered for editing
+    let toBeRendered = {} 
+    // Used for conversion of LC to long-form
+    let languageNames = new Intl.DisplayNames(['en'], {type: 'language'}); 
+
+    if (nodeObject) {
+        Object.keys(nodeObject).forEach((key) => {
+            if (key.startsWith('tags') && 
+                !key.includes('ids')) {
+                    languageCode = key.slice(-2);
+                    toBeRendered[languageCode] = nodeObject[key]
+                }
+        })
+    }
+
+    // Helper function used for changing state of properties
+    function changeDataComment(key, value) {
+        const duplicateData = {...nodeObject};
+        duplicateData[key] = value;
+        setNodeObject(duplicateData);
+    }
+
+    function changeData(key, index, obj) {
+        const duplicateData = {...nodeObject};
+        duplicateData[key][index] = obj;
+        setNodeObject(duplicateData);
+    }
+
+    return ( 
+        <div className="all-other-properties">
```

Comment:
nit: avoid having general classes, it gets tricky to maintain at some point. What about using <Box sx={{}} /> or some other Material UI option instead?

Reply:
@nobeeakon Would you be able to suggest any other alternatives?

Reply:
@aadarsh-ram , I have updated the previous comment, sorry for that... about other alternatives, mmm, I'm not completely familiar with the lastest version of Material, but they suggest some stuff in [here](https://mui.com/material-ui/customization/how-to-customize/)